temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -118,4 +118,41 @@ service CloudService {
             get: "/api/v1/regions/{region}",
         };
     }
+
+    // Gets all known nexus incoming services
+    rpc GetIncomingServices(GetIncomingServicesRequest) returns (GetIncomingServicesResponse) {
+	option (google.api.http) = {
+            get: "/api/v1/nexus/incomingservices",
```

Comment:
Open Question: Given incoming services are already scoped to an account, is there a need to have this "incoming services" path here? This felt more forward-facing, but can understand the argument to just have these hang off the top-level nexus path.

Reply:
I think it's better to be explicit and leave the `/nexus/` path open for other endpoints.